languages/ruby/bitwarden_sdk/lib/ruby-bitwarden-sdk.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'dry-types'
+
+require_relative 'schemas'
+require_relative 'extended_schemas/schemas'
+require_relative 'command_runner'
+require_relative 'bitwarden_lib'
+require_relative 'bitwarden_error'
+require_relative 'projects'
+require_relative 'secrets'
+
+module BitwardenSDK
+  class BitwardenClient
+    attr_reader :bitwarden, :project_client, :secrets_client
+
+    def initialize
+      client_settings = ClientSettings.new(
+        api_url: ENV['BITWARDEN_API_URL'] || 'https://api.bitwarden.com',
+        identity_url: ENV['BITWARDEN_IDENTITY_URL'] || 'https://identity.bitwarden.com/connect/token',
+        user_agent: 'Bitwarden RUBY-SDK',
+        device_type: nil
+      )
```

Comment:
I think we should follow the C# example and accept a optional settings object which can contain api_url and identity_url.

This makes us less opinionated and the consumers can configure it however they want. Note that we should not expose the `ClientSettings` but rather a new object.